test/test_dispatch.py

```diff
@@ -28,7 +27,7 @@ def test_mcmc_interface(model, backend):
         mcmc.summary()
 
 
-@pytest.mark.parametrize('backend', ['funsor', 'minipyro', 'numpy', 'pyro'])
+@pytest.mark.parametrize('backend', ['minipyro', 'numpy', 'pyro'])
```

Comment:
it took a lot of work to get these tests working, and it's a shame to see them go. is there a reason we can't simply fork the standard funsor backends into say {`funsor` = `funsor[torch]`, `funsor[numpy]`, `funsor[jax]`}?